poker/src/components/BlueButton/BlueButton.module.scss

```diff
@@ -0,0 +1,60 @@
+.BlueButton_blue,
+.BlueButton_white {
+  margin: 0 auto;
+  padding: 0 10px;
+  min-width: 240px;
+  border: 1px solid #496a81;
+  border-radius: 2px;
+  font-family: 'Ruda', sans-serif;
+  font-size: 1.5rem;
+  line-height: 2.5rem;
+  font-weight: bold;
+  transition: all 0.3s ease-in-out;
+  cursor: pointer;
+}
+
+.BlueButton_blue {
+  color: white;
```

Comment:
Bad naming: .BlueButton_blue  color: white;

Reply:
suggest something)

Reply:
change to GeneralButton